### PR TITLE
Introduce YTOP_PROXY env var

### PR DESCRIPTION
**Pull request by zlobober**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

This commit introduces YTOP_PROXY environment variable which brings back the possibility to run operator locally when cluster proxies have a well-known fqdn.

### CODE REVIEW

**Comment by savnadya** [comment]
Shipped!

File: pkg/components/ytsaurus_client.go [hunk 1]
```diff
@@ -518,8 +519,12 @@ func (yc *ytsaurusClient) doSync(ctx context.Context, dry bool) (ComponentStatus
 	if yc.ytClient == nil {
 		token, _ := yc.secret.GetValue(consts.TokenSecretKey)
 		timeout := time.Second * 10
+		proxy, ok := os.LookupEnv("YTOP_PROXY")
+		if !ok {
+			proxy = yc.cfgen.GetHTTPProxiesAddress(consts.DefaultHTTPProxyRole)
+		}
 		yc.ytClient, err = ythttp.NewClient(&yt.Config{
-			Proxy:               yc.cfgen.GetHTTPProxiesAddress(consts.DefaultHTTPProxyRole),
+			Proxy:               proxy,
```
Comment (gritukan) [inline pkg/components/ytsaurus_client.go:527]: Don't you want to enable DisableProxyDiscovery by default? Looks like it works unless some heavy commands are added into initialization pipeline.

Comment (zlobober) [reply pkg/components/ytsaurus_client.go:527]: Why won't we enable disable, indeed.

**Comment by savnadya** [comment]
Shipped!

**Pull request closed without merging**